lib/manageiq/release/internationalization.rb

```diff
@@ -0,0 +1,199 @@
+module ManageIQ
+  module Release
+    class Internationalization
+
+      attr_reader :branch, :dry_run
+
+      def initialize(branch: nil, dry_run: true)
+        @branch  = branch || 'master'
+        @dry_run = dry_run
+      end
+
+      def update_message_catalogs
+        klasses = []
+        klasses << ManageIQ
+        klasses << ManageIQ_ServiceUI
+
+        klasses.each do |klass|
+          klass.new(branch: branch, dry_run: dry_run).update_message_catalog
+        end
+      end
+
+      class Helper
+        def execute(command, description = nil)
+          description ||= "*** Running `#{command}` ***"
+          puts description
+          system(command)
+        end
+
+        def git_diff_stats(repo)
+          repo.git.capturing.diff(:numstat => true).split("\n").each_with_object({}) do |diff, stats|
+            insertions, deletions, fname = diff.split("\s")
+            stats[fname] = {:insertions => insertions.to_i, :deletions => deletions.to_i }
+          end
+        end
+
+        def create_tx_config(tx_config_content)
+          return if File.exist?(".tx/config")
+          puts "*** Creating .tx/config ***"
+          Dir.mkdir(".tx") unless Dir.exist?(".tx")
+          File.write(".tx/config", tx_config_content)
+        end
+
+        def with_checked_out_repo(repo, branch)
+          repo.fetch
+          repo.checkout(branch)
+          repo.chdir { yield }
+        end
+      end
+
+      class ManageIQ
+        attr_reader :branch, :dry_run, :helper
+
+        def initialize(branch: nil, dry_run: true)
+          @branch  = branch || 'master'
+          @dry_run = dry_run
+          @helper  = Helper.new
```

Comment:
Feels like Helper should be a base class, like `RepoBase`, and then `class ManageIQ < RepoBase`